Games/types/Mafia/effects/Stun.js

```diff
@@ -0,0 +1,44 @@
+const Effect = require("../Effect");
+const Action = require("../Action");
+const { PRIORITY_NIGHT_ROLE_BLOCKER } = require("../const/Priority");
+
+module.exports = class Stun extends Effect {
+
+    constructor(stunner) {
+        super("stunner");
```

Comment:
name is stun~

Reply:
?

Reply:
The effect name is "Stun", as you used in the Snowball file, it is a typo to use "stunner"

Reply:
Stunner is the player who applied the stun

Reply:
Oh, you're right about the super part